tests/apollo/test_skvbc_dbsnapshot.py

```diff
@@ -153,6 +156,55 @@ async def test_db_checkpoint_disabled(self, bft_network, tracker):
         num_of_db_snapshots =  await bft_network.get_metric(0, bft_network, "Counters", "numOfDbCheckpointsCreated", component="rocksdbCheckpoint")
         assert num_of_db_snapshots == 0
 
+    @with_trio
+    @with_bft_network(start_replica_cmd, selected_configs=lambda n, f, c: n == 7)
+    @verify_linearizability()
+    async def test_create_dbcheckpoint_cmd(self, bft_network, tracker):
+        """
+            sends a createdbCheckpoint command and test for created dbcheckpoints.
+        """
+        bft_network.start_all_replicas()
+        client = bft_network.random_client()
+        skvbc = kvbc.SimpleKVBCProtocol(bft_network, tracker)
+        for i in range(150):
+            await skvbc.send_write_kv_set()
+
+        op = operator.Operator(bft_network.config, client, bft_network.builddir)
+        rep = await op.create_dbcheckpoint_cmd()
+        rsi_rep = client.get_rsi_replies()
```

Comment:
get_rsi_replies returns response from all replicas, We should check individual response from replicas against the expected one

Reply:
fixed